taichi/backends/vulkan/vulkan_api.cpp

```diff
@@ -191,7 +192,8 @@ void ManagedVulkanDevice::create_instance(const Params &params) {
   app_info.applicationVersion = VK_MAKE_VERSION(1, 0, 0);
   app_info.pEngineName = "No Engine";
   app_info.engineVersion = VK_MAKE_VERSION(1, 0, 0);
-  app_info.apiVersion = params.api_version;  // important
+  app_info.apiVersion =
+      VK_API_VERSION_1_2;  // The highest version designed to use
```

Comment:
Please still use `params.api_version` and update the API version there. I'd like to keep `vulkan_api` itself as platform-neutral as possible.

Reply:
Actually what does params.api_version do? Is it the max API version to use or is it the exact API version used?

Reply:
>  is it the exact API version used

This